addons/hr_work_entry_contract/static/src/components/work_entry_source_field/work_entry_source_field.js

```diff
@@ -0,0 +1,36 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { evaluateExpr } from "@web/core/py_js/py";
+import { registry } from "@web/core/registry";
+import { RadioField, radioField } from "@web/views/fields/radio/radio_field";
+
+export class WorkEntrySourceField extends RadioField {
+    static template = "hr_work_entry_contract.WorkEntrySourceField";
+    static props = {
+        ...RadioField.props,
+    };
```

Comment:
No need, if we don't need to add new props. 🙂 

(since we extends of RadioField class, we will get the props defined inside that class)